worker/src/RTC/DtlsTransport.cpp

```diff
@@ -1335,6 +1335,10 @@ namespace RTC
 		  srtpRemoteMasterKey,
 		  srtpMasterLength,
 		  this->remoteCert);
+
+		delete[] srtpMaterial;
+		delete[] srtpLocalMasterKey;
+		delete[] srtpRemoteMasterKey;
```

Comment:
Amazing catch! so we were leaking right???

Reply:
Maybe we should use

```c++
uint8_t srtpMaterial[srtpMasterLength * 2];
```

instead of:

```c++
uint8_t* srtpMaterial = new uint8_t[srtpMasterLength * 2];
```

?

Reply:
Sorry, I don't understand, it seems that there was no leak at all. It happens that you have replaced:

```c++
uint8_t srtpMaterial[srtpMasterLength * 2];
```

with this:

```c++
uint8_t* srtpMaterial = new uint8_t[srtpMasterLength * 2];
// [...]
delete[] srtpMaterial;
```

Why? is it needed by Visual Studio? Doesn't it allocate arrays of variable length in the stack?

Reply:
yep.  just adapte to Visual Studio compiler.